src/gluonts/model/evaluation.py

```diff
@@ -94,9 +93,10 @@ def evaluate_forecasts_raw(
     test_data: TestData,
     metrics,
     axis: Optional[Union[int, tuple]] = None,
+    batch_size: int = 100,
     mask_invalid_label: bool = True,
     allow_nan_forecast: bool = False,
-    seasonality: Optional[int] = None
+    seasonality: int = 1
```

Comment:
Why is the default being changed to 1?

Reply:
Good point: I had initially done this because I was also computing the seasonal error in batches, that didn’t play well with potentially different seasonalities in a batch; turns out batching there is anyway problematic because of different length, so I should indeed be able to maintain the current behavior.

Reply:
Done